raw-svo.dict.lua

```diff
@@ -89,6 +89,80 @@ local dict_smoke_def = {}
 
 local codepaste = {}
 
+local tekura_stance_oncompleted = function (new_stance)
+  local stances = {
+    "horse", 
+    "eagle", 
+    "cat", 
+    "bear", 
+    "rat", 
+    "scorpion", 
+    "dragon"
+  }
+
+  for _, stance in ipairs(stances) do
+    defences.lost(stance)
+  end
+
+  defences.got(new_stance)
+end
+
+local tekura_stance_isadvisable = function (new_stance)
```

Comment:
This doesn't seem to be used for tekura forms only. Can you update the function name accordingly? Same with `shikudo_form_advisable` below.